module/userProjects/helper.js

```diff
@@ -1107,6 +1163,7 @@ module.exports = class UserProjectsHelper {
 					'hasAcceptedTAndC',
 					'link',
 					'entityId',
+					'acl',
```

Comment:
@Prajwal17Tunerlabs project template we are storing acl?

Reply:
I think you have misread the code @VISHNUDAS-tunerlabs , projectTemplate doesn't contain acl, the code is hidden while reviewing i guess.